x-pack/plugins/security_solution/public/overview/containers/overview_cti_links/helpers.ts

```diff
@@ -14,8 +14,8 @@ export interface CtiListItem {
   count: number;
 }
 
-export const EMPTY_LIST_ITEMS: CtiListItem[] = CTI_DEFAULT_SOURCES.map((item) => ({
-  title: item,
+export const EMPTY_LIST_ITEMS: CtiListItem[] = Object.keys(CTI_DEFAULT_SOURCES).map((title) => ({
```

Comment:
If the index signature typing above (`{ [key: string]: string }`) is for this `Object.keys` call , might I suggest doing something like [this](https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/common/search_strategy/security_solution/cti/index.ts#L38-L39) instead?